salt/elife-dashboard/scheduler.sls

```diff
@@ -8,8 +8,8 @@ install-{{ app.name }}:
         # note: elife-article-scheduler is always deployed as master
         # until it has its own instance, we cannot a revision
         # using build vars
```

Comment:
Does it mean if the dashboard itself is built on a different revision, the article scheduler always has to be built as master regardless?

Reply:
yes, at the moment it's always deployed as `master` (more like we do with libraries than with projects)

Reply:
there is a [ticket](https://elifesciences.atlassian.net/browse/ELPP-3111) to merge the two projects. I fancied the new python dev starting in Oct could take a look at it.